README.md

```diff
@@ -1,5 +1,17 @@
 **DISCLAIMER: THIS IS A WORK IN PROGRESS**
 
-# BigchainDB RFC
+# BigchainDB Enhancement Proposals (BEPs)
 
-This is the BigchainDB RFC project.
+This is the BigchainDB Enhancement Proposal project. We collect specifications for APIs, protocols, and processes.
+
+The process to add or change an EIP is the following:
+- A specification is created and modified by pull requests according to [C4](./1).
+- Specification lifecycle SHOULD follow the lifecycle defined in [COSS](./2).
+- Non-cosmetic changes are allowed only on [Raw](./2#raw-specifications) and [Draft](./2#draft-specifications) specifications.
+
+# Current BEPs
+
+Short Name | Title                                                        | Type     | Status     | Editor
+-----------|--------------------------------------------------------------|----------|------------|-------
+1/C4       | Collective Code Construction Contract                        | Meta     | Raw        | Alberto Granzotto
```

Comment:
Can we add links? At some point we might have more and then you need to scroll down to see a specific thing you're looking for and then scroll way (yes, at this point in time we have a lot) up to get there.

Reply:
Definitely yes, good point.